frontend/app/src/components/pincode/PincodeInput.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+    import { getContext, onMount } from "svelte";
+    import { type PincodeContext, type PincodeType } from "./Pincode.svelte";
+
+    export let value: string = "";
+    export let id: string = "input" + Math.random().toString(36);
+
+    let ref: HTMLInputElement | undefined;
+    let type: PincodeType;
+    let selectTextOnFocus: boolean;
+    let userAgent = navigator.userAgent;
+
+    const android = userAgent?.match(/android/i);
```

Comment:
Is there a capability based way of determining whether keydown will work?

Reply:
Not really sure that there is.